<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::ConnectionInspector</class>
 <widget class="QWidget" name="GammaRay::ConnectionInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="KFilterProxySearchLine" name="connectionSearchLine"/>
        </item>
        <item>
         <widget class="QTreeView" name="connectionView">
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kde/kfilterproxysearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
